apps/bluetooth-music/app.js

```diff
@@ -0,0 +1,98 @@
+'use strict'
+
+var logger = require('logger')('bluetooth-music')
+var Application = require('@yodaos/application').Application
+var bluetooth = require('@yoda/bluetooth')
+var protocol = bluetooth.protocol
+var AudioFocus = require('@yodaos/application').AudioFocus
+var audioFocus = null
+var a2dp = null
+
+function onAudioFocusGained () {
+  logger.debug('onAudioFocusGained')
+  a2dp.unmute()
+}
+
+function onAudioFocusLost () {
+  logger.debug('onAudioFocusLost')
+  a2dp.mute()
+  app.finalize()
```

Comment:
Where is the app declared?

Reply:
The `app` is declared below, and is the only exports of `Application`.